clear_cache.py

```diff
@@ -5,9 +5,9 @@
 
 db = getattr(settings, 'REDIS_DB', 0)
 if all(hasattr(settings, attr) for attr in
-    ['REDIS_MASTER_DNS', 'REDIS_PORT']):
+    ['REDIS_MASTER_DNS', 'RQ_DASHBOARD_REDIS_PORT']):
     conn = StrictRedis(host=settings.REDIS_MASTER_DNS,
-        port=settings.REDIS_PORT, db=db)
+        port=settings.RQ_DASHBOARD_REDIS_PORT, db=db)
```

Comment:
Do we need to change settings_lcoal.py correspondingly since "REDIS_PORT" is renamed to "RQ_DASHBOARD_REDIS_PORT"?  How the change config change in the settings_local.py affect the warnings in RQ?

Reply:
Is there an advantage in renaming this setting? For backwards compatibility (especially with various local_settings.py on developer machines), we would want to consider keeping the same variable name, if possible.

Reply:
I changed this setting because RQ dashboard will deprecate "REDIS_PORT" and other similar options soon, and they suggest add "RQ_DASHBOARD_" prefix to parameters.

To maintain the backward compatibility, I moved this part to another place so that it requires no change of `settings_local.py` on developers' machines or other repositories via [7b85934](https://github.com/bloomberg/pybossa/pull/648/commits/7b85934c810a96a2f5b82eaf58377f824bd42097)